### ISSUE TITLE
Tana-helper mac Application is not working when port 8000 is already occupied

### ISSUE THREAD

**Issue by doomuch** [verveguy/tana-helper#25]
<img width="237" alt="image" src="https://github.com/verveguy/tana-helper/assets/80600884/0387eb69-5d81-46a0-b12a-b657d86fb15e">
It does not allow to open WebUI

**Comment by verveguy** [comment]
Right. That's because the helper service fails to start if the port is already in use. I'll need to add a configuration option to the helper menu to change the port number. In the meantime, can you move the other service from port 8000 to something else?
